src/scripts/main.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve, reject) => {
```

Comment:
The `reject` parameter in the Promise executor function is not used. If you don't plan to use it, you can remove it to avoid confusion.